setup.py

```diff
@@ -76,6 +76,7 @@ def is_pure(self):
         "tensorflow-datasets",
         "keras-core",
         "kagglehub",
+        "keras_nlp",
```

Comment:
We will need to swap with `keras_nlp_nightly` if we are going to do this for nightly package.

Reply:
We don't want to add to `reqirements_common`. This will cause other issues since `keras_nlp` will install TF with it and we may want the GPU version. So lets keep `keras_nlp` out of requirements file.